GoogleForCreators/web-stories-wp
# Remove 'defaultProvider' and rename 'common' to 'pagination'

## Summary

The 'defaultProvider' parameter to each pagination action dispatcher is not clear. 

Also, the purpose of the 'common' folder is clearly pagination only, and so can be named as such to make this clearer.

This PR has the following changes:
- Removes defaultProvider.
- Renames 'common' folder to 'pagination'.

Additional improvements may be made to state management as development progresses and we explore further.

## Testing Instructions

- Open the local media pane and scroll to the next page.
- Upload an image and verify.
- Upload a video and verify.
- Open the media3p pane and scroll to the next page.

---



Addresses #3317
